src/main/scala/io/sdkman/changelogs/GcnMigrations.scala

```diff
@@ -0,0 +1,49 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "080")
```

Comment:
There is an error in other migration and `079` is duplicated. So, one of those should move to `080`.

```suggestion
@ChangeLog(order = "081")
```